docs:userFlow 수정 및 비즈니스로직 변경 #20

hsp64 opened the pull request:
# 📄 Documentation Pull Request
> 제목 규칙: **[Docs]: 문서 내용 요약 (#이슈번호)**  
> 예: [Docs]: 프로젝트 구조도 업데이트 (#8)

# 20 docs:userFlow 수정 및 비즈니스로직 변경
---

## PR 유형 (Type of PR)
> 해당하는 항목에 `x` 표기해주세요.
- [ ] 문서 추가 (New Docs)
- [x] 문서 수정 (Update Docs)
- [x] 오타/형식 수정 (Fix Typo / Format)
- [ ] 기타 (Chore)

---

## PR 요약 (PR Summary)
> 이 PR이 어떤 문서에 어떤 변경을 했는지 간단히 설명해주세요.

(여기에 내용 작성)

- 파일명 변경 userflow_md.md -> userFlow.md
- .gitkeep 임시 파일 삭제
- 신고 기능 SHOULD 로 변경(기존 MUST)
- 3.1 AI챗봇 위치 변경(좌하단 -> 우하단)
- 6.1 모임 상태정의 모집완료 -> 모집마감으로 변경
---

## 관련 이슈 (Related Issue)
> 이 PR과 연결된 이슈 번호가 있다면 작성해주세요.

- Closes #
- Resolves #

- #20 docs:userFlow 수정 및 비즈니스로직 변경
---

## 변경 사항 (Changes)
> 변경된 문서 목록과 주요 내용을 구체적으로 적어주세요.

- docs/README.md: 프로젝트 소개 보강
- docs/architecture.md: 서비스 다이어그램 업데이트

---

## 스크린샷 / 미리보기 (Preview)
> 문서의 렌더링 결과나 다이어그램이 있다면 첨부해주세요.

(여기에 파일 첨부)

---

## 셀프 체크리스트 (Self-Checklist)
- [x] 제목 규칙을 지켰습니다.
- [ ] 문서 빌드/렌더링 시 문제가 없습니다.
- [x] 맞춤법/링크 오류를 확인했습니다.
- [x] 관련 이슈를 연결했습니다.

---

## 리뷰어에게 (To Reviewers)
> 리뷰 시 유의해야 할 부분, 확인이 필요한 특정 내용이 있다면 작성해주세요.

오타(파일명) 및 챗봇 위치 수정과 신고버튼(should) 변경했습니다.

먼저 회의 내용 반영했습니다. 비즈니스 로직 MUST에 따른 300시작 100점 500점 시 배지변경과 신고 버튼 SHOULD 로 변경(mermaid에만 명시)했고 이탈자, 노쇼, 개인 점수 작업 악용 등으로 생성: 2점, 참여:1점, 피드백 +-3점, 노쇼 -20 으로 정했습니다.

추가적으로 회원당 방생성 제한을 두는 방법( 점수 작업 방지 및 수익아이템 )은 향후 계획 때 다시 이야기 하겠습니다. 감사합니다.

- 278번줄 수정했습니다, 추가적으로 메인페이지 구조 회원가입 삭제 /  참여하기 버튼 검색창에 바꿧습니다. (동현님 와이어프레임도 반영 예정)

hsp64 committed:
docs:userFlow 수정 및 비즈니스로직 변경
- 파일명 변경 userflow_md.md -> userFlow.md
- .gitkeep 임시 파일 삭제
- 신고 기능 SHOULD 로 변경(기존 MUST)
- 3.1 AI챗봇 위치 변경(좌하단 -> 우하단)
- 6.1 모임 상태정의 모집완료 -> 모집마감으로 변경

minee0505 reviewed (commented):
등급 시스템 278줄 다시 적어주세요

minee0505 reviewed (changes requested):
등급 시스템 278줄 수정 부탁드립니다.

hsp64 committed:
docs:userFlow 수정 및 비즈니스로직 변경
- 파일명 변경 userflow_md.md -> userFlow.md
- .gitkeep 임시 파일 삭제
- 신고 기능 SHOULD 로 변경(기존 MUST)
- 3.1 AI챗봇 위치 변경(좌하단 -> 우하단)
- 6.1 모임 상태정의 모집완료 -> 모집마감으로 변경

- 278번줄 등급 시스템 수정

minee0505 reviewed (approved):
수고하셨습니다!
